src/common.cc

```diff
@@ -81,7 +82,9 @@ common::Error::Error(const stim::DemInstruction& error) {
 }
 
 std::string common::Error::str() const {
-  return "Error{cost=" + std::to_string(likelihood_cost) + ", symptom=" + symptom.str() + "}";
+  std::stringstream ss;
+  ss << std::fixed << std::setprecision(2) << likelihood_cost;
```

Comment:
@draganaurosgrbic any chance we can revert this `std::setprecision(2)`? I think just `std::to_string(likelihood_cost)` is best so that a user can see exactly what the error is being assumed by the decoder.